src/tests/ftest/io/NvmeIO.yaml

```diff
@@ -0,0 +1,49 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+
+hosts:
+ servers:
+  - boro-A
+  - boro-B
+  - boro-C
+  - boro-D
+ clients:
+  - boro-E
+timeout: 28800
```

Comment:
Does this test really take that long?

Reply:
Yes, There are some large IOR sizes tests in sequence.

Reply:
If possible we can look to increate the clients to bring run time lower.